woocommerce/woocommerce
# CYS: Revert Zoom Feature

### Changes proposed in this Pull Request:

We recognized that the zoom-out view isn't fully ready. Some interactions need to be revisited, and certain copy updates are required.

Considering that:
* Today is the last day of the cycle and the next cycle will start when the code freeze has already started
* We’re still determining the next steps for this project.
* We believe it’s best not to include this feature in WooCommerce 9.3.

The following PRs were reverted:
* https://github.com/woocommerce/woocommerce/pull/50070
* https://github.com/woocommerce/woocommerce/pull/50411
* https://github.com/woocommerce/woocommerce/pull/50436



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure your CYS experience has been reset.
2. Enter the customize your store flow (non-AI).
3. Check that the resizer handler is present, and the device toolbar with the zoom features no longer exists.
4. Check that the resizer handler works as expected.
5. Check for no regressions introduced by this PR.
6. Build and upload the plugin to an ecom plan on WP.com and check that non of these changes are present in the AI flow.
7. Ensure there are no regressions in the Launch Your Store flow since we updated some shared logic.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
